packages/salesforcedx-vscode-apex/src/languageServer.ts

```diff
@@ -91,8 +91,7 @@ async function createServer(
 
     return {
       options: {
-        env: process.env,
-        stdio: 'pipe'
```

Comment:
if you remove the pipe, does that mean output goes to the console now?

Reply:
Version 9 does not seem to accept that property anymore.